Jenkinsfile_CNP

```diff
@@ -67,23 +67,23 @@ withPipeline(type, product, component) {
   }
 
   before('smoketest:preview') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.preview.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.service.core-compute-preview.internal/"
   }
 
   afterAlways('smoketest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
   }
 
   before('smoketest:aat') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend.aat.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-staging.service.core-compute-aat.internal/"
   }
 
   afterAlways('smoketest:aat') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
   }
 
   before('functionalTest:preview') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.preview.platform.hmcts.net/"
+    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend-pr-${CHANGE_ID}.service.core-compute-preview.internal/"
```

Comment:
this url should be - https://prl-citizen-frontend-pr-886.preview.platform.hmcts.net

So line change 86 is correct